src/Spryker/Client/Cart/Plugin/SessionQuoteStorageStrategyPlugin.php

```diff
@@ -206,6 +207,60 @@ public function changeItemQuantity($sku, $groupKey = null, $quantity = 1)
         return $this->increaseItemQuantity($sku, $groupKey, $delta);
     }
 
+    /**
+     * Specification:
+     * - Makes zed request.
```

Comment:
Zed
please also check other occurrences.

Reply:
Backport is not that important, please check this on the main master releases.

Reply:
ok, thx